uvicorn/protocols/http/httpunk_impl.py

```
@@ -0,0 +1,489 @@
+            pending = None
+            streaming.set_result(True)
+
+        async def receive() -> dict[str, Any]:
+            if st["complete"]:
+                return {"type": "http.disconnect"}  # pragma: no cover
+            try:
+                chunk = await body_iter.__anext__()
+                return {"type": "http.request", "body": chunk, "more_body": True}
+            except StopAsyncIteration:
+                return {"type": "http.request", "body": b"", "more_body": False}
+            except Exception:  # pragma: no cover
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)